tests/examples/test_examples.py

```diff
@@ -115,6 +115,10 @@ def test_mlflow_run_example(directory, params, tmpdir):
     cli_run_list = [tmp_example_dir] + params
     invoke_cli_runner(cli.run, cli_run_list)
 
+    if get_free_disk_space() < 5.0:  # unit: GiB
+        stdout = process.exec_cmd(["./dev/remove-conda-envs.sh"])[1]
+        print(stdout)
+
```

Comment:
`remove-conda-env.sh` removes all existing conda envs, and then run `conda remove --all`:

https://github.com/mlflow/mlflow/blob/master/dev/remove-conda-envs.sh